liberu-genealogy/genealogy-old
# Issue/social login

# Description

In this PR,  Make api social login, This is will resolve issue #1576 

# Checklist

- [x] add code from old backend repository
- [x] test with POSTMAN

# How to test on POSTMAN

```
provider: Github
Method: GET
Url : http://localhost:8000/api/login/github

Callback from github
"https://github.com/login/oauth/authorize?client_id=de95cdcbf6819c63786d&redirect_uri=http%3A%2F%2F127.0.0.1%3A8000%2Flogin%2Fgithub%2Fcallback&scope=user%3Aemail&response_type=code"

Use Callback
you will see page login into github
```

# Screenshots

Result from url app
![Screenshot 2021-08-25 at 01 42 28](https://user-images.githubusercontent.com/43591658/130664332-536f0431-7990-4d2f-b884-72135a039b7c.png)

Result from github
![Screenshot 2021-08-25 at 01 42 35](https://user-images.githubusercontent.com/43591658/130664505-7ebac913-a7a4-4d77-82c7-2d905d3411c2.png)
